src/client/Microsoft.Identity.Client/AppConfig/BaseAbstractApplicationBuilder.cs

```diff
@@ -218,13 +217,6 @@ internal void ResolveAuthority()
                 // Both WithAuthority and WithTenant were used at app config level
                 if (!string.IsNullOrEmpty(Config.TenantId))
                 {
-                    if (!Config.Authority.AuthorityInfo.CanBeTenanted)
```

Comment:
Replace with a log?

Reply:
We don't have a logger at this stage, since the app is only being built.